Makefile

```diff
@@ -354,6 +354,7 @@ fv fv/latency.log fv/data-races.log: $(REMOTE_DEPS) image-test bin/iptables-lock
 	  FV_K8SIMAGE=$(FV_K8SIMAGE) \
 	  FV_NUM_BATCHES=$(FV_NUM_BATCHES) \
 	  FV_BATCHES_TO_RUN="$(FV_BATCHES_TO_RUN)" \
+	  FELIX_FV_FELIX_LOG_LEVEL="$(FELIX_FV_FELIX_LOG_LEVEL)" \
```

Comment:
Don't think we need two FELIX here, and `FV_FELIX_LOG_LEVEL` would look more like other settings.